### PR TITLE
Update images from circleci to cimg

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
These image updates were the ones that were fairly straightforward (I think anyway). 

# Reasons
Legacy images are being replaced by convenience images, and we are replacing where appropriate.

### CODE REVIEW

**Comment by devinmarieb** [comment]
For whoever reviews this, there are only a few commits that matter to the images specifically (the others were to resolve build/pipeline issues and to add tabs for code blocks). These are the commits that need to be reviewed for the image updates:

- Update images from circleci to cimg _(further updates on this page based on clarification in the following two commits)_
- Legacy image swap for postgres
- Update adv-config.md

File: jekyll/_cci2/adv-config.md [hunk 1]
```diff
@@ -58,6 +126,57 @@ For more information on using shell scripts in your config, see the [Using Shell
...
+jobs:
+  build:
+    docker:
+      - image: cimg/node-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run: mkdir test-reports
```
Comment (felicianotech) [inline jekyll/_cci2/adv-config.md:144]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/adv-config.md [hunk 2]
```diff
@@ -58,6 +126,57 @@ For more information on using shell scripts in your config, see the [Using Shell
...
```
Comment (felicianotech) [inline jekyll/_cci2/adv-config.md:169]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/adv-config.md [hunk 3]
```diff
@@ -58,6 +126,57 @@ For more information on using shell scripts in your config, see the [Using Shell
...
+{:.tab.executors_three.Server_3}
+```yaml
+version: 2.1
+
+orbs: 
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/node-browsers
```
Comment (felicianotech) [inline jekyll/_cci2/adv-config.md:163]: This is missing a tag. Something like:

`- image: cimg/node:17.0-browsers`

File: jekyll/_cci2/adv-config.md [hunk 4]
```diff
@@ -87,8 +206,96 @@ For more information on browser testing, see the [Browser Testing]({{site.baseur
...
+    # Service container image
+      - image: cimg/postgres:9.6.5
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+
+    steps:
+      - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install postgresql-client-9.6
```
Comment (felicianotech) [inline jekyll/_cci2/adv-config.md:236]: If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`

File: jekyll/_cci2/pipeline-variables.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/node:latest
+      - image: cimg/node:latest
```
Comment (felicianotech) [inline jekyll/_cci2/pipeline-variables.md:48]: latest is not a tag

maybe :17.0

**Review by felicianotech — CHANGES_REQUESTED** [review]

**Comment by devinmarieb** [comment]
@felicianotech I have made the changes and double checked for additional places that had the same needs in these files. Thank you!

**Review by felicianotech — APPROVED** [review]

**Pull request merged**